templates/navbar.php

```diff
@@ -1,5 +1,47 @@
-<?php 
+<?php
+$loggedIn = true;
+/**
+ * admin
+ * director
+ */
+$role = 'director';
 
-echo "navbar";
+?>
+<header class="container">
+    <div class="row">
+        <div class="nav_logo">
+            <img src="../public/assets/img/logo.png" alt="Logo And action" class="logo">
+        </div>
+        <ul class="nav_list">
+            <li class="nav_list_item"><a class="nav_link" href="#">Home</a></li>
+            <?php
+            if ($role == 'admin') {
+            ?>
+                <li class="nav_list_item"><a class="nav_link" href="#">Daschboard</a></li>
```

Comment:
Spelfoutje op lijn 20 en lijn 27

Reply:
Klopt het dat een normale gebruiker (wanneer ingelogd), die **niet** de rol 'director' heeft nu alleen de home button en Sign out button heeft? Dit haal ik uit de code. Zie mock-up afbeelding voor normale gebruiker

![image](https://user-images.githubusercontent.com/89914058/144128569-e2b996e7-30ae-4421-b9e0-74bf056c22ab.png)

Reply:
Klopt, de normale gebruiker ziet alleen home en sign out en account(maar dit komt later). De mock-up voor dezez gebruiker is er niet, het enigste wat anders was waren de items van de nav